packages/features/webhooks/lib/scheduleTrigger.ts

```diff
@@ -438,8 +453,20 @@ export async function updateTriggerForExistingBookings(
 
   if (addedEventTriggers.length > 0) {
     const promise = bookings.map((booking) => {
-      return addedEventTriggers.map((triggerEvent) => {
-        scheduleTrigger({ booking, subscriberUrl: webhook.subscriberUrl, subscriber: webhook, triggerEvent });
+      return addedEventTriggers.map(async (triggerEvent) => {
+        const { booking: bookingWithWebhookPayload, evt } = await getBooking(booking.id);
+        const webhookData = getWebhookPayloadForBooking({
+          booking: bookingWithWebhookPayload,
+          evt,
+        });
```

Comment:
Can we move the fetching of the booking and creation of the `webhookData` outside `addedEventTriggers.map`?

Reply:
Sure, thanks for catching that.